README.md

```diff
@@ -50,4 +50,26 @@ Currently there's no vendoring provided for this project.
 * **vulcan-unclassified** - Example vulnerability to test the monitoring of unclassified vulnerabilities - not for production
 * **vulcan-vulners** - Runs https://vulners.com/api/v3/burp/software/
 * **vulcan-wpscan** - Checks Wordpress sites for vulnerabilities using the open source wpscan utility
-* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
\ No newline at end of file
+* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
+
+## Building and testing
+
+This project is primarily built using the [vulcan-checks-bsys](https://github.com/adevinta/vulcan-checks-bsys) project.
+
+But it's possible to build each one of the checks with `go build`.
+
+In every check directory there is an example configuration file called `local.toml.example`, most checks reads a
+file named `local.toml` if you pass the `-t` parameter, so by copying that file you can do a local test of your code
+before you commit.
+
+The checks that are not self-contained, and instead relay on external binaries, can be locally tested with
+`vulcan-checks-bsys`. By running `vulcan-build-images -r PATH_TO_THE_CHECK_DIR` it will create a docker image and
+run it, feeding the check the input from local.toml.
+
+Full example to build and test run one check:
+```
+cd cmd/vulcan-drupal
+go build
+cp local.toml.example local.toml
+./vulcan-drupal -t
```

Comment:
I suggest adding here also the alternative option:
```
go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal 
```

Reply:
`go run` doesn't seem to support that, I'm getting:
```
$ go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal 
package github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
```

Reply:
Yeah! it's new from Go 1.17 :)

```
$ cd cmd/vulcan-drupal
$ go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:447: Reading manifest for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:470: Running go build for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:453: Building image for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:465: Docker image built, build log:
...
```

For older versions you should use `go install` or `go get`, depending on the version also.

Reply:
Added information about `go run` together with the caveat about go version

Reply:
very cool, thx! :)